package_manifest.yaml

```diff
@@ -1,20 +1,104 @@
----
-packages:
-  children:
-    foreman_packages: {}
-    plugins_packages: {}
-    katello_packages: {}
-    katello_client_packages: {}
-
 foreman_packages:
-  vars:
```

Comment:
I also liked vars above hosts because it's usually shorter and sort of a preset for the rest.